lib/kits/core-ui/components/paged-publication/main-container.js

```diff
@@ -154,6 +153,20 @@ const MainContainer = ({
             e.stopPropagation();
             sgnPp?.parentNode?.removeChild(sgnPp);
         });
+
+        sgnPp.addEventListener(
```

Comment:
It's important when doing escape logic that the current view always has focus() and tabindex="-1". So when you close the menu, it's important you focus() the publication viewer both so that arrows work for navigation and that escape works.